### PR TITLE
Drools 2082 static code analysis

### PR DESCRIPTION
**Pull request by jomarko**
Prepare `drools-wb-verifier` module for the static code analysis that is already turned on in some other projects of kiegroup.

@manstis please have a look. 99% of changes is maked up from licences and imports organization. I think just in one file I have added `{}` for if statements.

### CODE REVIEW

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/JavaCheckRunner.java [hunk 1]
```diff
@@ -1,9 +1,24 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/JavaCheckRunner.java]: 2017 (if I was being nit-picking)

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/PriorityListCheck.java [hunk 2]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/PriorityListCheck.java]: 2017 (if I was being nit-picking)

**Comment by jomarko** [comment]
looking on git history seems to me as 2016

**Comment by manstis** [comment]
@jomarko Quite possibly, I did not check. Given (1) there are lots of classes in the module with a date of 2016, (2) it's a petty, trivial observation; I merged "as is"... but I don't doubt the creation date was 2016!

**Pull request merged**